docs/community/governance.md

```diff
@@ -3,6 +3,8 @@
 This page provides links to documents for common Knative community practices and
```

Comment:
same comment here on sidebar vs page title: "Community Rules and Practices" vs "About the community" (github wouldn't let me tag the right line though...)

Reply:
Updated the title on this doc, as the nav title was better.